extensions/machine-learning-services/src/modelManagement/interfaces.ts

```diff
@@ -49,12 +49,13 @@ export type WorkspacesModelsResponse = ListWorkspaceModelsResult & {
  * An interface representing registered model
  */
 export interface RegisteredModel {
-	id?: number,
-	artifactName?: string,
-	title?: string,
-	created?: string,
-	version?: string
-	description?: string
+	id?: number;
+	artifactName?: string;
+	title?: string;
+	created?: string;
+	version?: string;
+	description?: string;
+	filePath?: string;
```

Comment:
hmm... I think this is we what we already discussed around no required fields... but should at least the ones we show like ID/artifact should be required to ensure our table looks okay.

Reply:
yeah I'll create separate models. this module is used for storing models (that don't have Id yet) and to load models so that's why everything is optional here. but I can improve it

---
In reply to: [389191872](https://github.com/microsoft/azuredatastudio/pull/9450#discussion_r389191872) [](ancestors = 389191872)